src/graph/query_graph.c

```diff
@@ -427,7 +565,7 @@ GrB_Matrix QueryGraph_MatrixRepresentation(const QueryGraph *qg) {
 
 	GrB_Matrix m;   // Matrix representation of QueryGraph.
 	GrB_Info res = GrB_Matrix_new(&m, GrB_BOOL, node_count, node_count);
-	assert(res == GrB_SUCCESS);
+	ASSERT(res == GrB_SUCCESS);
```

Comment:
This line causes a compiler warning:
`warning: variable ‘res’ set but not used`
The typical way to squash this is by adding the line:
`(void)res;`
Or a macro to do the same.

Reply:
👍